[docs] Add 'Integrations' section (#712)
gitpod-io/website: src/docs/github-integration.md

# GitHub Integration

> Gitpod comes with a [GitHub](https://github.com/) integration that allows you to start workspace from a GitHub project.

Author: ```suggestion
Gitpod comes with a [GitHub](https://github.com/) integration that allows you to start Gitpod workspaces from any GitHub project.
```

## Starting Workspaces

To start a workspace from a GitHub project, prefix the GitHub URL with `gitpod.io/#` as described in [Getting Started](/docs/getting-started/).

## Enabling Prebuilds

> To enable [prebuilt workspaces](/docs/prebuilds/) for your GitHub project you need to install the [Gitpod GitHub app ](https://github.com/apps/gitpod-io) to the GitHub account the projects belongs to. Install the app as described on the [prebuilds page](/docs/prebuilds/#on-github).

Author: ```suggestion
To enable [prebuilt workspaces](/docs/prebuilds/) for your GitHub project you need to install the [Gitpod GitHub app](https://github.com/apps/gitpod-io) and configure it for the GitHub user or organization that the project belongs to. Install the app as described in the [prebuilds page](/docs/prebuilds/#on-github).
```
